config/locales/referee_interface/referee_interface.yml

```diff
@@ -50,7 +50,7 @@ en:
   reference_confirmation_email:
     subject: Reference submitted for %{candidate_name}
   reference_cancelled_email:
-    subject: Reference request cancelled by %{candidate_name}
+    subject: %{candidate_name} no longer needs a reference
```

Comment:
```suggestion
    subject: "%{candidate_name} no longer needs a reference"
```

YAML values that start with a special character like `%` need to be quoted like this I think. Weirdly the `%` is fine if it's not the first character.

Reply:
ahhh. thanks Steve!